From ba16e2554e5d02de10f1b22ece7c200ba5899557 Mon Sep 17 00:00:00 2001
From: Naveen M K <naveen521kk@gmail.com>
Date: Sun, 25 Jun 2023 17:20:52 +0530
Subject: [PATCH 092/N] Don't convert `sysconfig.get_config_var('VPATH')` to
 an absolute path

this is required for venv to work properly
---
 Makefile.pre.in |  1 -
 configure.ac    | 10 +---------
 2 files changed, 1 insertion(+), 10 deletions(-)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index cfaa0e3..98fc780 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -146,7 +146,6 @@ datarootdir=    @datarootdir@
 
 # Locations needed for semi-native fixup of sysconfig.
 srcdir_b2h=		@srcdir_b2h@
-VPATH_b2h=		@VPATH_b2h@
 abs_srcdir_b2h=		@abs_srcdir_b2h@
 abs_builddir_b2h=	@abs_builddir_b2h@
 prefix_b2h=		@prefix_b2h@
diff --git a/configure.ac b/configure.ac
index 13ba8e8..b91b48f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -763,12 +763,8 @@ fi
 # without setting up the same env and also that the build of Python
 # itself will fail as the paths are not correct for the host tools.
 #
-# Also, getpath.c uses GetModuleFileNameW (replacing \ with /) and
-# compares that with the define VPATH (passed in via command-line)
-# to determine whether it's the build- or the installed-Python.
-#
 # To work around these issues a set of _b2h variables are created:
-# VPATH_b2h, prefix_b2h, srcdir_b2h, abs_srcdir_b2h
+# prefix_b2h, srcdir_b2h, abs_srcdir_b2h
 # and abs_builddir_b2h
 # .. where b2h stands for build to host. sysconfig.py replaces path
 # prefixes matching the non-b2h versions with the b2h equivalents.
@@ -795,10 +791,6 @@ AC_DEFUN([ABS_PATH_HOST],
 AC_SUBST([$1])
 ])
 
-AC_MSG_CHECKING(absolute host location of VPATH)
-ABS_PATH_HOST([VPATH_b2h],[srcdir])
-AC_MSG_RESULT([$VPATH_b2h])
-
 AC_MSG_CHECKING(absolute host location of prefix)
 ABS_PATH_HOST([prefix_b2h],[prefix])
 AC_MSG_RESULT([$prefix_b2h])
